src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
+}
```

Comment:
Fields of enum should be in Upper-case (BLACK, WHITE...) because they are **CONSTANTS**. <br/>
You can read about naming rules [here](https://mate-academy.github.io/style-guides/java/java.html#s5.1-identifier-names).